ga4gh/backend.py

```diff
@@ -9,6 +9,7 @@
 import os
 
 import ga4gh.protocol as protocol
+from ga4gh.frontend import app
```

Comment:
I guess we'd want to fill out the config dependent stuff in the frontend. We'll have to do a bit of thinking to see what the best approach is, but I'm sure there's a simple solution which preserves the frontend/backend split.